app/javascript/flavours/glitch/components/status.js

```diff
@@ -466,6 +466,7 @@ export default class Status extends ImmutablePureComponent {
               onOpenVideo={this.handleOpenVideo}
               width={this.props.cachedMediaWidth}
               cacheWidth={this.props.cacheMediaWidth}
+              revealed={settings.getIn(['media', 'reveal_behind_cw']) ? !!status.get('spoiler_text') : undefined}
```

Comment:
I think this would interfere with the always show sensitive media option. You should use `true` if both conditions are met, but `undefined` otherwise!

Reply:
you are correct, will fix

Reply:
pushed fix